oci-test/tests/oci_streaming_stream/test-get-expected.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "{{ output.resource_id.value }}",
+    "lifecycle_state": "ACTIVE",
```

Comment:
There is attribute reference for lifecycle_state https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/streaming_stream#lifecycle_state_details mostly we need to avoid the hardcoded value as much as possible, please give a try.